passes/S038/README.md

```diff
@@ -0,0 +1,86 @@
+# S038
+
+The S038 analyzer reports cases of Schemas which include `ConflictsWith` and have invalid schema attribute references.
+
+NOTE: This pass only works with Terraform resources that are fully defined in a single function.
```

Comment:
Terraform resources that meet the criteria for this check can come from a variety of source code, e.g. variables, inline declarations in `schema.Provider.ResourcesMap`, etc.

```suggestion
NOTE: This pass only works with `schema.Resource` that are wholly declared.
```

Reply:
You're right